Group_Facilitator_Name.ipynb

```diff
@@ -425,7 +425,7 @@
     "# Check for possible columns with outliers\n",
     "#\n",
     "numerical_cols = [\n",
-    "    'loan_amnt', 'int_rate', 'installment', 'annual_inc', 'dti', 'revol_util', 'open_acc', 'total_acc'\n",
```

Comment:
Is this Column causing issue?

Reply:
'revol_util' has NaN values which was causing the crash